File: COMP_HW1/COMP_HW1_java/Friend.java
public class Friend extends Person{
	private int age;
	
	public Friend(String a, String b, int c, int n) {
		super(a, b, c);
		age = n;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}
	
	public void print(){
		super.print();
		System.out.print("_" + age);
	}
}

File: COMP_HW1/COMP_HW1_cpp/Friend.cpp
/*
 * Friend.cpp
 *
 *  Created on: 2016. 4. 20.
 *      Author: USER
 */

#include "Friend.h"
using namespace std;

Friend::Friend(string a, string b, int c, int d) : Person(a, b, c){
  age = d;
}

void Friend::setAge(int a){
  age = a;
}

int Friend::getAge(){
  return age;
}

void Friend::print(){
  Person::print();
  cout << "_" << age;
}


